### ISSUE TITLE
Crash on macOS arm64

### ISSUE THREAD

**Issue by joeyparrish** [shaka-project/static-ffmpeg-binaries#51]
macOS arm64 builds of FFmpeg are currently crashing on GitHub VMs when testing Shaka Streamer.

**Comment by joeyparrish** [comment]
Builds made on the GitHub VM crash on the system that built them.

A debug build (`-g -O0`) also crashes.

`lldb` shows this when it crashes:

```
* thread #5, name = 'av:h264:df3', stop reason = EXC_BAD_ACCESS (code=1, address=0x0)
    frame #0: 0x0000000100498454 ffmpeg`ff_h264_idct_add16intra_neon + 40
ffmpeg`ff_h264_idct_add16intra_neon:
->  0x100498454 <+40>: ldrb   w3, [x7], #0x1
    0x100498458 <+44>: ldrsw  x0, [x5], #0x4
    0x10049845c <+48>: ldrb   w3, [x4, w3, uxtw]
    0x100498460 <+52>: add    x0, x0, x6
Target 0: (ffmpeg) stopped.
```


Configuring with `--disable-neon` failed to build.

**Comment by joeyparrish** [comment]
The M1 mac in the lab couldn't build ffmpeg at all with these instructions.  The error was about "illegal text relocations" in some assembly functions for aarch64.  I'm not sure why this is different from the build environment of the GitHub VMs.

I couldn't quite solve this, until I found a reference to `--enable-pic` being required for iOS builds.  Since position independent code (PIC) is related to relocation (that's my very surface level understanding), I tried configuring with `--enable-pic`, and that got the lab machine to build ffmpeg.  The resulting build did not crash on the same machine.

I'm trying `--enable-pic` now on the GitHub VMs to see if this helps fix the crash.
